### PR TITLE
Change memcached dependency to a specific version of it

### PR DESCRIPTION
**Pull request by fortizpenaloza**

### CODE REVIEW

**Comment by fortizpenaloza** [comment]
@gcotelli I think we can do a release after it is approved. A new major version?

**Review by gcotelli — DISMISSED** [review]

File: source/BaselineOfSuperluminal/BaselineOfSuperluminal.class.st [hunk 1]
```diff
@@ -57,7 +57,7 @@ BaselineOfSuperluminal >> setUpDependencies: spec [
 		project: 'Teachable-Core' copyFrom: 'Teachable' with: [ spec loads: 'Core' ].
 		
 	spec 
-		baseline: 'Memcached' with: [ spec repository: 'github://svenvc/memcached:master' ];
+		baseline: 'Memcached' with: [ spec repository: 'github://svenvc/memcached:v1.0.0' ];
 		project: 'Memcached-Core' copyFrom: 'Memcached' with: [ spec loads: 'Memcached' ]
```
Comment (gcotelli) [inline source/BaselineOfSuperluminal/BaselineOfSuperluminal.class.st]: We can use `Deployment` group here instead of a package name.

**Comment by gcotelli** [comment]
> @gcotelli I think we can do a release after it is approved. A new major version?

Yes, we need to release v2.0.0 and write the migration guide or provide a migration package with automatic rewrite rules if we can.

File: source/BaselineOfSuperluminal/BaselineOfSuperluminal.class.st (hunk 1 above)
Comment (fortizpenaloza) [reply source/BaselineOfSuperluminal/BaselineOfSuperluminal.class.st]: Yes, we can. I will fix this right away!

**Review by gcotelli — APPROVED** [review]

**Pull request merged**